requirements/core.txt

```diff
@@ -13,7 +13,7 @@ requests>=2.22.0
 scikit-learn>=1.0.1
 scipy>=1.3.2
 shap>=0.40.0
-statsforecast>=1.0.0
+statsforecast>=1.4.0
```

Comment:
Are you sure this is the lowest possible version that supports everything?
I'm double checking because the higher our lower bounds, the more stringent a condition it puts on users who may have constraints on their Pypi settings.

Reply:
I'm going to look into it for a bit more. I think it might depend on if we want AutoCES to be probabilistic as well. But let me check it and I'll come back to you with the options.

Reply:
@hrzn, I'm sure now that it is as follows:

 - ETS (from statsforecast 1.3.2 onwards is called AutoETS) is available from 1.0.0, but only made probabilistic in 1.4.0
 - AutoTheta became available in 1.3.0 and was probabilistic from the start
 - AutoCES became available in 1.1.0 and is not yet probabilistic but will be -soon- in the next pip release (1.4.1/1.5.0?)

So if we don't choose 1.4.0 as a requirement, then we will have to drop the probabilistic nature of StatsForecastETS. 
And we might want to rename StatsForecastETS to StatsForecastAutoETS, because it is also renamed by statsforecast. 

What would you like me to put in there as a requirement? And do you want me to rename the ETS model?

Reply:
OK thanks a lot for investigating @Beerstabr .
We can keep 1.4.0 as the lower bound (and even update it once AutoCES is made probabilistic). I think that's OK because anyway all releases of statsforecast are quite recent.
I also think we should rename `StatsForecastETS` to `StatsForecastAutoETS` if they renamed it on their side.
Thanks!